### ISSUE TITLE
'Form does not validate'

### ISSUE THREAD

**Issue by zcsongor** [JamesGuthrie/angular-schema-form-signature#2]
Even if I fill out the canvas with signature, the error message says 'Form does not validate' therefore I can't submit my form. Do you have any idea why this happens?

Thanks!

**Comment by JamesGuthrie** [comment]
Unfortunately I don't know why this happens. I have occasionally seen this behaviour, and been able to get it to work by resetting and then trying again. Is it happening all of the time?

**Comment by zcsongor** [comment]
Yeah, it works sometimes and it doesn't once in a while. Going to give it a go with the new release :)

**Comment by JamesGuthrie** [comment]
I don't think that the new release will make a difference with that unfortunately :/ If I could figure out how to reproduce the bug it would make it a lot easier to fix...

**Comment by zcsongor** [comment]
I see. Trying to figure it out myself as well, will update this thread if I found something.
